public/js/components/AtomEdit/AtomEdit.js

```diff
@@ -40,8 +38,13 @@ class AtomEdit extends React.Component {
   }
 
   updateAtom = (newAtom) => {
-    this.props.atomActions.updateAtom(newAtom);
-    enterPresence(this.props.routeParams.atomType, this.props.routeParams.id);
+    try {
+        enterPresence(this.props.routeParams.atomType, this.props.routeParams.id);
+    } catch (e) {
+      console.warn(e); // eslint-disable-line no-console
```

Comment:
To get around this there's a `logger` module included. You can see examples in the action creators. e.g. https://github.com/guardian/atom-workshop/blob/master/public/js/actions/AtomActions/createAtom.js#L4